css/styles.css

```diff
@@ -0,0 +1,157 @@
+* {
+	background-color: #FFFFFF;
+	font-family: sans-serif;
+}
+
+body {
```

Comment:
@humphriescarly We typically wouldn't apply a width to the body, although it clearly works and your page, might I add, looks really great! I would probably do something like this instead:
```html
<body>
  <div class="container">
    <!-- All my page content would go here inside the container div. -->
    <!-- I would style this div with a width and center it on the page. -->
  </div>
</body>
```

Now I would use your css to style the container div instead of the body:
```css
.container {
  width: 800px;
  margin: 0 auto;
}